src/etc/inc/unbound.inc

```diff
@@ -518,13 +519,25 @@ function unbound_add_host_entries() {
             if ($host['host'] != "")
                 $host['host'] = $host['host'].".";
             if (!$added_item[$current_host]) {
-                $host_entries .= "local-data-ptr: \"{$host['ip']} {$host['host']}{$host['domain']}\"\n";
-                if (is_ipaddrv6($host['ip']))
-                    $host_entries .= "local-data: \"{$host['host']}{$host['domain']} IN AAAA {$host['ip']}\"\n";
-                else
-                    $host_entries .= "local-data: \"{$host['host']}{$host['domain']} IN A {$host['ip']}\"\n";
-                if (!empty($host['descr']) && isset($config['unbound']['txtsupport']))
+                /* Backwards compatibility for records created before introducing RR types. */
+                if (!isset($host['rr'])) {
+                    $host['rr'] = (is_ipaddrv6($host['ip'])) ? 'AAAA' : 'A';
```

Comment:
The idea was, that the user does not need to care about the difference between `A` and `AAAA` records, since without this PR the user does not need to care about it neither. So the differentiation between A and AAAA is done programmatically and is shown as "A/AAAA".
I have no problem in separating A and AAAA RRs...